packages/ui/src/app/pages/Profile/MyAccounts.tsx

```diff
@@ -52,6 +53,7 @@ export const MyAccounts = () => {
   const { total, transferable, locked, recoverable, vestingTotal, vestedClaimable, vestingLocked } =
     useMyTotalBalances()
   const { hasAccounts, isLoading } = useMyAccounts()
+  const [shouldDismissBanner, setShouldDismissBanner] = useState(false)
```

Comment:
Based on this:
![image](https://github.com/Joystream/pioneer/assets/6571453/5c18518d-cfd0-43ed-93ba-410396cc7025)

The dismissed information should be stored in the localstorage instead of the component state:
```suggestion
  const [shouldDismissBanner, setShouldDismissBanner] = useLocalStorage<boolean>('buy-joy-banner') ?? false
```